recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
```

Comment:
IMHO it would be nice to keep this separate if this is an option. That way someone has the option of running this locally if needed.

What do you think @msarahan and @patricksnape?